### ISSUE TITLE
Issue with k8s.io/zh/docs/concepts/policy/

### ISSUE THREAD

**Issue by webf5** [kubernetes/website#32134]
`https://kubernetes.io/zh/docs/concepts/policy/pod-security-policy/` and 'https://kubernetes.io/zh/docs/tutorials/clusters/apparmor/'
is unable to access

**Comment by tengqm** [comment]
The former page is being fixed (#32128)
The latter one is an outdated link.  The link is mentioned in the following pages:

```
docs/tasks/configure-pod-container/security-context.md: needs resync.
docs/concepts/policy/pod-security-policy.md: bad link, fixed in #32138
docs/tutorials/_index.md: needs resync.
docs/reference/command-line-tools-reference/feature-gates.md: needs resync.
```

**Comment by Sea-n** [comment]
The latter is available in the new URL.
https://kubernetes.io/zh/docs/tutorials/security/apparmor/

Close as completed.
/close
